app/javascript/utils/map.js

```diff
@@ -0,0 +1,32 @@
+export const getPolygonCenter = polygon => {
```

Comment:
In this case, it might be worth using turf (you can import similar to lodash) http://turfjs.org/docs/#center. If this works then great, but for pease of mind a supported library might be more robust.